### ISSUE TITLE
mlocktweaker locking only lowercase modes with charybdis

### ISSUE THREAD

**Issue by KoraggKnightWolf** [atheme/atheme#646]
Dear Atheme Developers,

It appears that on version 7.3-r1 with mlocktweaker it only locks all modes in lowercase but not the uppercase ones. 
Example, Charybdis 4.1.1 and Atheme 7.3-r1: 
 mlocktweak {  "+Cnt"; }; is in the config
Expected result: mlock would be +Cnt-lk 
Actual result: mlock is +cnt-lk 
(the +C and +c channel mode for Charybdis are both loaded)

This is confirmed to occur with the aforementioned IRCd, untested on others, I hope this can help in debugging or explaining to me what the issue is.

Regards,

Koragg

**Comment by aaronmdjones** [comment]
I am unable to reproduce this issue; I can successfully instruct services to mlock both `+C` and `+O` on channel registration:

```
loadmodule "modules/contrib/mlocktweaker";

chanserv {
	nick = "ChanServ";
	user = "ChanServ";
	host = "services.arpa";
	real = "Channel Services";
	mlocktweak = "+COnt";
};
```

```
05:39 -!- Aaron [aaron@localhost] has joined #foo
05:39 -!- ServerMode/#foo [+nt] by hades.arpa
05:39 [Users #foo]
05:39 [@Aaron] 
05:39 -!- Irssi: #foo: Total of 1 nicks [1 ops, 0 halfops, 0 voices, 0 normal]
05:39 -!- Channel #foo created Sun Dec 30 05:39:47 2018
05:39 -!- Irssi: Join to #foo was synced in 0 secs
```

```
05:40 -!- Irssi: Starting query in TestNet with ChanServ
05:40 <Aaron> REGISTER #foo
05:40 -ChanServ(ChanServ@services.arpa)- #foo is now registered to Aaron.
```

```
05:40 -!- ChanServ [ChanServ@services.arpa] has joined #foo
05:40 -!- ServerMode/#foo [+o ChanServ] by services.arpa
05:40 -!- mode/#foo [+OC] by ChanServ
```

```
05:43 <Aaron> INFO #foo
05:43 -ChanServ(ChanServ@services.arpa)- Information on #foo:
05:43 -ChanServ(ChanServ@services.arpa)- Founder    : Aaron
05:43 -ChanServ(ChanServ@services.arpa)- Successor  : (none)
05:43 -ChanServ(ChanServ@services.arpa)- Registered : Dec 30 05:39:59 2018 +0000 (3m 0s ago)
05:43 -ChanServ(ChanServ@services.arpa)- Mode lock  : +ntOC-lk
05:43 -ChanServ(ChanServ@services.arpa)- Flags      : VERBOSE KEEPTOPIC GUARD PRIVATE PUBACL
05:43 -ChanServ(ChanServ@services.arpa)- *** End of Info ***
```



Are you sure you rehashed services after modifying the configuration file?

**Comment by KoraggKnightWolf** [comment]
Dear @aaronmdjones, 

Thank You very much for taking the time and effort in attempting to reproduce the bug I encountered. It appears it was a falsely utlized syntax on my end in my atheme.conf for mlocktweaker. 
I had used: mlocktweak { "+Cnt"; };
While your usage was: mlocktweak = "+COnt";
Upon utilizing the correct syntax, the bug no longer occurs, thus I will close this issue and would like to thank You once again for Your time and effort. 

Regards,

Koragg
